js/live-editor.js

```diff
@@ -65,37 +66,8 @@ window.LiveEditor = Backbone.View.extend({
         this.record = new ScratchpadRecord();
 
         // Set up the Canvas drawing area
-        this.drawCanvas = new ScratchpadDrawCanvas({
-            el: this.dom.DRAW_CANVAS,
-            record: this.record
-        });
-
-        this.drawCanvas.on({
-            // Drawing has started
-            drawStarted: function() {
-                // Activate the canvas
-                this.$el.find(this.dom.DRAW_CANVAS).show();
-            }.bind(this),
-
-            // Drawing has ended
-            drawEnded: function() {
-                // Hide the canvas
-                this.$el.find(this.dom.DRAW_CANVAS).hide();
-            }.bind(this),
-
-            // A color has been chosen
-            colorSet: function(color) {
-                // Deactivate all the color buttons
-                this.$el.find(this.dom.DRAW_COLOR_BUTTONS)
-                    .removeClass("ui-state-active");
-
-                // If a new color has actually been chosen
-                if (color !== null) {
-                    // Select that color and activate the button
-                    this.$el.find("#" + color).addClass("ui-state-active");
-                }
-            }.bind(this)
-        });
+        // should only do this if there's a flag set
+        //this.setupDrawCanvas();
```

Comment:
I was trying to see how quickly I could get the page to load for the case where a student is working on a program in the live-editor.  There are additional pieces of code that are packaged with live-editor but aren't always used: playback code used when students watch a lesson and recording code used when you create a lesson.  I was thinking that page load speeds could be improved if we only load these components when they're required.  Not loading these components also improves memory use.